push_sender.js

```diff
@@ -33,7 +33,7 @@ mq.on('message', function(message) {
   ).catch((err) => {
     if (err.statusCode === 410) { // subscription gone
       const id = message.subscription.id;
-      debug(`Subscription #${id} deleted (was gone)`);
+      debug('Subscription #%s deleted (was gone)', id);
```

Comment:
maybe we don't care, but there is `%d` for integers. I think the different is that when we pass an object, %s will call `toString` while %d will call `valueOf`. So not sure we care after all.